### DISCUSSION TITLE
console.log found uncrypted password.

### DISCUSSION THREAD

**Discussion by Toei79** [GerryDazoo/Slinger#381] [General]
so i able to fix mine, then i found password unencrypted in the console.log

there was encrypted one idk what it was starting for EI: that never worked in my case

checking more in the console.log  , i found the password.

i found the password in my case 8aUh0plqPhSiQAx

**Comment by socelillyhammer** [comment]
@GerryDazoo this is correct I can confirm this finding.  If you use my method in #374 or have your log you can get the decripted password in the console.log file as well.

Just run a search for **"USER":"admin","PASSWORD"** and you will see the decrypted password in the console.log file

**Reply by sp222222** [reply]
good stuff my man. and to think I was going to totally ignore you and toe's conversion in the other disscussions. (these come at me as emails constantly, so any comment made on any thread I am in will send me the email).

pure gold that find !
